pyanaconda/core/configuration/payload.py

```diff
@@ -111,3 +111,8 @@ def verify_ssl(self):
         this option.
         """
         return self._get_option("verify_ssl", bool)
+
+    @property
+    def import_keys_to_rpm(self):
+        """Import keys into RPM database at end of installation."""
+        return self._get_option("import_keys_to_rpm", str)
```

Comment:
It should be a list of keys. Please, call `split()` on the result.

Reply:
Done.